src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/CoreType.cs

```diff
@@ -72,7 +72,7 @@ internal enum CoreType
 
     internal static class CoreTypeHelpers
     {
-        public static void GetFullName(this CoreType coreType, out byte[] ns, out byte[] name)
+        public static void GetFullName(this CoreType coreType, out ReadOnlySpan<byte> ns, out ReadOnlySpan<byte> name)
```

Comment:
L124 should be changed to use `ns = name = default` rather than `ns = name = null`

Reply:
Done.